src/Engine/Component/Physics.hpp

```diff
@@ -61,5 +60,14 @@ namespace Component {
              */
             glm::vec3 momentOfInertia = glm::vec3(1.f, 1.f, 1.f);
 
+        public:
```

Comment:
Two public modifiers.

Reply:
The idea here is that when managers fully creates components in the future this would become private. I'll remove it for now until this is in place.